.github/workflows/test.yml

```
@@ -24,8 +24,12 @@ jobs:
           cache: 'npm'
           registry-url: 'https://registry.npmjs.org'
 
-      - name: Build with Node.js ${{ matrix.node }} on ${{ matrix.os }}
-        run: npm install && npm run compile
+      - name: Install zstd
+        run: npm run install-zstd
+        shell: bash
+
+      - name: install dependencies and compmile
+        run: npm install --loglevel verbose
```

```suggestion
        run: npm install
```

I don't feel too strongly do we always want this logging at this level or was this left over from debugging?

---

fine by me